src/main/java/org/elasticsearch/action/admin/cluster/snapshots/delete/TransportDeleteSnapshotAction.java

```diff
@@ -64,7 +64,7 @@ protected DeleteSnapshotResponse newResponse() {
 
     @Override
     protected ClusterBlockException checkBlock(DeleteSnapshotRequest request, ClusterState state) {
-        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA, "");
+        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA_WRITE, "");
```

Comment:
@imotov what do you think here? METADATA_READ or METADATA_WRITE? is it the same problem as described above?